.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
- [ ] Gitignore is for ignoring files and not uploading them into the repositories. Although you have added the test.md in your gitignore, I still can see that in your project directory. Kindly fix this.
![image](https://user-images.githubusercontent.com/91486791/235139382-b5d0d252-f228-4236-9494-d2ea6aa78508.png)